lib/app.js

```diff
@@ -61,6 +62,13 @@ app.post('/cloud/:datasetId', function(req, res) {
 var excludedPaths = ['/authpolicy', '/file'];
 app.use(require('fh-wfm-user/lib/middleware/validateSession')(mediator, mbaasApi,excludedPaths));
 
+
+//Initialising the fh-wfm-sync data sets for each module
+wfmSync.init(mediator, mbaasApi, 'workorders', config.get("syncOptions"));
```

Comment:
Not sure how it would make setup any easier. It's one function call with config for that data set.

It's the same principle as the $fh.sync framework uses.